tests/integration/devfile/cmd_dev_test.go

```diff
@@ -275,8 +279,9 @@ var _ = Describe("odo dev command tests", func() {
 				})
 			})
 
-			When("running odo delete component --wait", func() {
+			When("killing odo dev and running odo delete component --wait", func() {
```

Comment:
Don't you think this test is more appropriate for cmd_delete_test.go?
Or, to keep it more inline with the following test, check instead that it does not delete the resources.?

Reply:
I find interesting to place it here as we can see the similarity with the test `when stopping odo dev normally`